apps/condo/domains/property/components/panels/Builder/BuildingPanelEdit.tsx

```diff
@@ -279,6 +279,10 @@ export const BuildingPanelEdit: React.FC<IBuildingPanelEditProps> = (props) => {
     }, [id, push])
 
     const menuClick = useCallback((event) => {
+        if (INSTANT_ACTIONS.includes(event.key)) {
+            Map[event.key]()
```

Comment:
Naming convention that is used for `Map` is better to use for classes. Class instance is an object, so `mapEdit` would be better here.